bindings/c/test/mako/mako.cpp

```diff
@@ -184,10 +184,7 @@ int populate(Database db,
 
 	Transaction systemTx = db.createTransaction();
 	for (int i = 0; i < args.total_tenants; ++i) {
-		while (true) {
-			// Until this issue https://github.com/apple/foundationdb/issues/7260 is resolved
-			// we have to commit each tenant creation transaction one-by-one
-			// while (i % 10 == 9 || i == args.total_tenants - 1) {
+		while (i % 10 == 9 || i == args.total_tenants - 1) {
```

Comment:
If I'm understanding this correctly, it's only going to create every 10th tenant (plus the last one)? I think this needs to be structured something like the following to create all of them and properly retry on errors:

```
for (int i = 0; i < args.total_tenants; i += batchSize) {
    while (true) {
        for (int j = 0; j < batchSize && i+j < args.total_tenants; ++j) {
            // create tenant i+j
        }
        // try to commit
        // break if successful
    }
}
```

Also, to get up to 1M tenants we'll probably need a larger batch size. I suspect we could easily do 1K in a single batch, and probably 10K would work too.

Reply:
I guess I was looking at an out-of-date version of this code.